plugins/module_utils/dnac.py

```diff
@@ -485,6 +486,30 @@ def update_site_type_key(self, config):
 
         return new_config
 
+    def is_valid_ip(self, ip_address):
```

Comment:
Can we change into ipv4?

Reply:
Hi I didn't keep the name IPv4 since we don't use ipv6 as internal ip address. Even on GUI we only use IPv4